tests/ArticlesPage.js

```diff
@@ -3,74 +3,81 @@ import CutePage from "../page-objects/pages/CutePage";
 import SubscribePage from "../page-objects/pages/SubscribePage";
 const { default: HomePage } = require("../page-objects/pages/Homepage");
 
-describe("Articles pages", () => {
+describe("Check the widgets on the articles page", () => {
```

Comment:
I think this test is for widgets functionality. Maybe can specify this into the description